pkg/keys/ed25519.go

```diff
@@ -32,8 +32,15 @@ type ed25519Verifier struct {
 	key       *data.PublicKey
 }
 
-func (e *ed25519Verifier) Public() string {
-	return string(e.PublicKey)
+func (e *ed25519Verifier) Public() (string, error) {
+	publicKey, err := json.Marshal(ed25519Verifier{
```

Comment:
I think this can be reverted.

Reply:
Reopening—I think we want to keep returning just `e.PublicKey` and not the marshalled version